akka-docs/rst/java/code/docs/testkit/ParentChildTest.java

```diff
@@ -45,7 +45,7 @@ public Receive createReceive() {
     public Receive createReceive() {
       return receiveBuilder()
         .matchEquals("ping", message -> {
-          getContext().parent().tell("pong", self());
+          getContext().getParent().tell("pong", self());
```

Comment:
My 5cts as a dual developer (Scala and Java) I would expect accessors to use the get/set pattern, only for Java of course! I think it may come from the habit of years of writing zillions of beans...